docker/dev/build.sh

```diff
@@ -37,3 +67,23 @@ docker build -f ${DOCKERFILE} \
     --build-arg VERSION=${VERSION} \
     --build-arg COMMIT=${COMMIT} \
     ${DOCKER_BUILD_ARGS} ../..
+
+echo "To push to repository please use command:"
```

Comment:
I would prefer not to put info about tag/pushing in build script for the image. That's needed only needed in deployment which we do quite differently.

Reply:
Well I remove that. I will created the push_image.sh instead which directly does pushing.